src/openrct2/ride/Vehicle.h

```diff
@@ -372,6 +375,25 @@ enum : uint32_t
     VEHICLE_ENTRY_FLAG_DODGEM_CAR_PLACEMENT = 1u << 31,
 };
 
+constexpr const uint8_t ANIMATION_SPEED_MAX_EXPONENT = 32;
+constexpr const uint8_t STEAM_EFFECT_TRANSLATION_COEFFICIENT = 32;
+
+const uint8_t VehicleDefaultAnimationSpeeds[] = {
+    // first 5 bits: bitshift value where 31 is slowest, 0 is fastest
+    // last 3 bits: multiplier 0-7 (be warned a multiplier of 0 means animations will not play)
+    // Exponent value in JSON objects is 32 - bitshift, such that "Exponent":12 translates to bitshift value of 20
```

Comment:
This comment should be above the declaration, in an /* */ block.

Otherwise: good job on annotating and commenting on it! :+1: